bin/gobblin.sh

```diff
@@ -135,6 +137,7 @@ function print_gobblin_service_usage() {
     echo "    --job-conf-file <job-conf-file-path>  Only for mapreduce mode: configuration file for the job to run"
     echo "    --help                                Display this help."
     echo "    --verbose                             Display full command used to start the process."
+    echo "    --args                                Additional -D args"
```

Comment:
"args" can be confusing since its basically a java arguments
btw isnt this covered as jvmopts ? since any strings passed as jvmopts will be placed in java command.

Reply:
Ah true I missed that, thanks for pointing it out!

Reply:
@jhsenjaliya I looked through the scripts again, you mean `jvmflags` not `jvmopts` right? Because the argument parsing seems to only accept `jvmflags`

Reply:
actually thats the typo in the help text `--jvmopts`, and I meant to keep the jvmopts since that is more intutive, let me fix that since its some more variable changes and send PR, in meantime, you can assume to pass it as `jvmopts`  Thanks